src/msw/textctrl.cpp

```diff
@@ -2809,7 +2809,7 @@ bool wxTextCtrl::MSWOnNotify(int idCtrl, WXLPARAM lParam, WXLPARAM *result)
                     int x = GET_X_LPARAM(msgf->lParam),
                         y = GET_Y_LPARAM(msgf->lParam);
 
-                    HandleMouseEvent(msg, x, y, flags);
+                    HandleMouseEvent(msg, x, y, flags, ::GetMessageExtraInfo());
```

Comment:
These changes could be avoided if we called `::GetMessageExtraInfo()` in `HandleMouseEvent()` itself too.